keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ApproveOldVersionTests.kt

```diff
@@ -59,7 +62,12 @@ abstract class ApproveOldVersionTests<T : KeelRepository> : JUnit5Minutests {
 
     val environment: Environment = Environment(
       name = "test",
-      constraints = setOf(ManualJudgementConstraint())
+      constraints = setOf(ManualJudgementConstraint()),
+      resources = setOf(
+        resource(
+          spec = DummyArtifactReferenceResourceSpec(artifactReference = "my-artifact")
+        )
+      )
```

Comment:
had to add actual resources to a couple of tests that exercise constraint evaluation